File: examples/cache/delete/go/deleteAValue.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/cache"
)

// Delete a value from the cache. If key not found a success response is returned.
func DeleteAvalue() {
	cacheService := cache.NewCacheService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := cacheService.Delete(&cache.DeleteRequest{
		Key: "foo",
	})
	fmt.Println(rsp, err)
}

File: examples/js/cache/delete/deleteAValue.js
const { CacheService } = require("m3o/cache");

// Delete a value from the cache
async function deleteAvalue() {
  let cacheService = new CacheService(process.env.MICRO_API_TOKEN);
  let rsp = await cacheService.delete({
    key: "foo",
  });
  console.log(rsp);
}

deleteAvalue();
